### ISSUE TITLE
Toast messages positioning needs to be fixed

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/server#33796]
![Screenshot 2022-09-01 at 17 45 26](https://user-images.githubusercontent.com/3404133/187956895-b572d508-9778-462a-a6d4-6964b2f6e4cd.png)

**Comment by jancborchardt** [comment]
@juliushaertl I assume this is just for reference / for the record, and you have the adjustments we talked about during our call? (Basically just positioning slightly to left and border-radius-large :)

**Comment by juliushaertl** [comment]
Yes, this might need work on the nextcloud-dialogs wrapper that we have as npm package

**Comment by szaimen** [comment]
@skjnldsv wasn't this already addressed?
